dts/bindings/sensor/st,lis2dux12-common.yaml

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2023 STMicroelectronics
+# SPDX-License-Identifier: Apache-2.0
+
+include: sensor-device.yaml
+
+properties:
+  irq-gpios:
+    type: phandle-array
+    description: |
+      DRDY pin
+
+      This pin defaults to active high when produced by the sensor.
+      The property value should ensure the flags properly describe
+      the signal that is presented to the driver.
+
+  range:
+    type: int
+    default: 0
+    description: |
+      Range in g. Default is power-up configuration.
+
+    enum:
+      - 3  # 16g (0.488 mg/LSB)
+      - 2  #  8g (0.244 mg/LSB)
+      - 1  #  4g (0.122 mg/LSB)
+      - 0  #  2g (0.061 mg/LSB)
+
+  power-mode:
+    type: int
+    default: 0
+    description: |
+      Specify the sensor power mode. Default is power-down mode
+
+    enum:
+      - 0 # Power Down (PD)
+      - 1 # Low Power (LP)
+      - 2 # High Resolution (HR)
+      - 3 # High Frequency (HF)
```

Comment:
here you can create a include/zephyr/dt-binding/sensor file and use values in DT. See example already in main